### PR TITLE
Add missing website demos

### PR DESCRIPTION
**Pull request by xintongxia**
#### Background
For v7.0 release, fix and add missing embed demos.

#### Change List
- Add ColumnLayerDemo, GridCellDemo, S2LayerDemo

ColumnLayerDemo
Screen Shot 2019-04-01 at 5 10 18 PM

GridCellDemo
Screen Shot 2019-04-01 at 5 10 56 PM

S2LayerDemo
Screen Shot 2019-04-01 at 5 10 42 PM

### CODE REVIEW

File: docs/layers/column-layer.md [hunk 1]
```diff
@@ -20,20 +21,21 @@ const App = ({data, viewport}) => {
...
+    radius: 250,
+    angle: Math.PI / 2,
     extruded: true,
-    getPosition: d => d.position,
-    getColor: d => d.color,
-    getElevation: d => d.elevation
+    pickable: true,
+    getPosition: d => d.centroid,
+    getColor: d => [48, 128, d.value * 255, 255],
+    getElevation: d => d.value * 5000
```
Comment (Pessimistress) [inline docs/layers/column-layer.md]: Use `elevationScale` for the constant?

File: docs/layers/column-layer.md [hunk 2]
```diff
@@ -20,20 +21,21 @@ const App = ({data, viewport}) => {
...
-    radius: 500,
-    angle: 0,
+    radius: 250,
+    angle: Math.PI / 2,
     extruded: true,
-    getPosition: d => d.position,
-    getColor: d => d.color,
-    getElevation: d => d.elevation
+    pickable: true,
+    getPosition: d => d.centroid,
```
Comment (Pessimistress) [inline docs/layers/column-layer.md]: `centroid` or `position`? Should match the data sample

File: docs/layers/column-layer.md [hunk 3]
```diff
@@ -20,20 +21,21 @@ const App = ({data, viewport}) => {
...
    * ]
    */
   const layer = new ColumnLayer({
     id: 'column-layer',
     data,
     diskResolution: 12,
-    radius: 500,
-    angle: 0,
+    radius: 250,
+    angle: Math.PI / 2,
```
Comment (Pessimistress) [inline docs/layers/column-layer.md]: Is this necessary?

File: docs/layers/grid-cell-layer.md [hunk 4]
```diff
@@ -30,11 +32,12 @@ const App = ({data, viewport}) => {
...
     id: 'grid-cell-layer',
     data,
-    cellSize: 500,
+    pickable: true,
     extruded: true,
-    getPosition: d => d.position,
-    getColor: d => d.color,
-    getElevation: d => d.elevation
+    cellSize: 200,
+    getPosition: d => d.centroid,
```
Comment (Pessimistress) [inline docs/layers/grid-cell-layer.md]: Ditto

File: docs/layers/s2-layer.md [hunk 5]
```diff
@@ -22,32 +24,26 @@ const App = ({data, viewport}) => {
...
-    wireframe: true,
-    lineWidthMinPixels: 1,
+    extruded: true,
     getS2Token: d => d.token,
-    getElevation: d => d.population / d.area / 10,
-    getFillColor: d => [d.population / d.area / 60, 140, 0],
-    getLineColor: [80, 80, 80],
-    getLineWidth: 1,
+    getFillColor: d => [d.value * 255, (1 - d.value) * 255, (1 - d.value) * 128, 128],
+    getElevation: d => Math.random() * 1000,
```
Comment (Pessimistress) [inline docs/layers/s2-layer.md]: Use `value`?

File: docs/layers/s2-layer.md [hunk 6]
```diff
@@ -22,32 +24,27 @@ const App = ({data, viewport}) => {
    *   {
    *     // S2 Cell in SF Bay Area
    *     token: "80858004",
-   *     zipcode: 94107,
-   *     population: 26599,
-   *     area: 6.11
+   *     value: 0.5979242952642347:
```
Comment (Pessimistress) [inline docs/layers/s2-layer.md]: Extra `:`

File: examples/layer-browser/package.json [hunk 7]
```diff
@@ -10,7 +10,7 @@
   },
   "dependencies": {
     "@loaders.gl/gltf": "^0.6.1",
-    "@luma.gl/debug": "^7.0.0-alpha",
+    "@luma.gl/debug": "7.0.0-alpha.24",
```
Comment (Pessimistress) [inline examples/layer-browser/package.json]: Don't lock the version. I think the latest is beta.1 now.

File: docs/layers/s2-layer.md [hunk 8]
```diff
@@ -22,32 +24,27 @@ const App = ({data, viewport}) => {
...
+    elevationScale: 1000,
     getS2Token: d => d.token,
-    getElevation: d => d.population / d.area / 10,
-    getFillColor: d => [d.population / d.area / 60, 140, 0],
-    getLineColor: [80, 80, 80],
-    getLineWidth: 1,
+    getFillColor: d => [d.value * 255, (1 - d.value) * 255, (1 - d.value) * 128, 128],
+    getElevation: d => d.value,
     onHover: ({object, x, y}) => {
       const tooltip = `${object.zipcode}\nPopulation: ${object.population}`;
```
Comment (xintongxia) [inline docs/layers/s2-layer.md]: make sure tooltip match between docs and code

**Pull request merged**